### PR TITLE
feat(landing-page): redesigned footer with improved appearance, functionality, and countdown section

### PR DESCRIPTION
**Pull request by NerioNitharshan**

### CODE REVIEW

File: package-lock.json
Comment (BirnadinErick) [inline package-lock.json:1]: we use pnpm, please delete this

File: api/v1/controllers/Projects.php
Comment (BirnadinErick) [inline api/v1/controllers/Projects.php:1]: somehow this push is different from what we've seen on the meeting

File: tailwind.config.cjs
Comment (BirnadinErick) [inline tailwind.config.cjs:1]: please re-confirm change in these colors.

**Review by BirnadinErick — CHANGES_REQUESTED** [review]

**Comment by NerioNitharshan** [comment]
Okay

File: package-lock.json
Comment (NerioNitharshan) [reply package-lock.json:1]: done

File: tailwind.config.cjs
Comment (NerioNitharshan) [reply tailwind.config.cjs:1]: no change till now

**Pull request closed without merging**